core/channel.go

```diff
@@ -50,12 +56,17 @@ func CreateChannel(src, dst *ProvableChain, ordered bool, to time.Duration) erro
 		// In the case of failure, increment the failures counter and exit if this is the 3rd failure
 		case !chanSteps.Success():
 			failures++
-			log.Printf("retrying transaction...")
+			logger.Info("retrying transaction...")
 			time.Sleep(5 * time.Second)
 			if failures > 2 {
+				logger.Error(
+					"! Channel failed",
+					err,
+				)
 				return fmt.Errorf("! Channel failed: [%s]chan{%s}port{%s} -> [%s]chan{%s}port{%s}",
 					src.ChainID(), src.Path().ClientID, src.Path().ChannelID,
-					dst.ChainID(), dst.Path().ClientID, dst.Path().ChannelID)
+					dst.ChainID(), dst.Path().ClientID, dst.Path().ChannelID,
```

Comment:
Looks like the arguments are incorrect.
Is the following correct instead?
```
src.ChainID(), src.Path().ChannelID, src.Path().PortID 
dst.ChainID(), dst.Path().ChannelID, dst.Path().PortID 
```

Reply:
@bluele You are right, even though this is not caused by this PR.

Reply:
@dongrie May I ask you fix this, too?

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/46c50e288b43464120b99c7ba2e3299afd21410b
Fix incorrect log